src/SignalR/server/Core/src/HubConnectionContext.cs

```diff
@@ -358,6 +362,13 @@ private async Task WriteHandshakeResponseAsync(HandshakeResponseMessage message)
         /// Aborts the connection.
         /// </summary>
         public virtual void Abort()
+        {
+            // REVIEW: Should we provide a new API that *doesn't* prevent automatic reconnects?
+            _allowAutomaticReconnect = false;
+            AbortAllowAutoReconnect();
```

Comment:
nit: this name isn't great, it doesn't really do anything specific to allowing reconnect

Reply:
What would you call this?

Reply:
AbortInner?

Reply:
That feels lazy to me. Might as well call it AbortCore(). At least this name indicates how it's different from a normal Abort().